main.py

```diff
@@ -103,6 +150,28 @@ def enable_pipeline(self, pipeline_id):
         log.debug(f"enable_pipeline result {msg} 201")
         return jsonify(msg), 200
 
+    def emit_event(self, name, content=None):
+        """Send event"""
+        context="kytos/of_multi_table"
+        event_name = f"{context}.{name}"
+        event = KytosEvent(name=event_name, content=content)
+        self.controller.buffers.app.put(event)
+
+    @listen_to("kytos/(mef_eline|coloring|of_lldp).enable_table")
+    def on_enable_table(self, event):
+        """Listen for NApps responses"""
+        self.handle_enable_table(event)
+        
+    def handle_enable_table(self, event):
+        """Handle NApps responses from enable_table"""
+        self.event = event
+        napp = event.name.split('/')[1].split('.')[0]
+        self.required_napps.remove(napp)
```

Comment:
I have a concern here. Do I check what is in the content and check what I have inside `self.current_enabled_table`? For example:
- `mef_eline` is running late with the response of a first attempt to enable a pipeline.
- The user enables a second pipeline (resetting the values from `self.required_napps`) with different set up for `mef_eline`.

Is it possible that `of_multi_table` receives the NApps' responses from the second enabling plus the response from `mef_eline` first enabling? I could have another variable `self.current_table_groups` with the last table groups to check content of responses.

Reply:
Regarding `self.required_napps`, `self.current_enabled_table` and `self.current_table_groups`. Notice that you're modelling them as instance level global without being aware of pipeline, since we'll allow only a single pipeline to be enabled then it wouldn't cause many issues to debug. I'll encourage you to take a step back, and think they all are even needed in the first place, based on [how to enable a pipeline](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP036.rst#enabling-pipeline), `self.current_enabled_table` and `self.current_table_groups` don't seem necessary, between steps 3 and 4, when disabling the current (soon to be previous) pipeline then you can get it and then store it locally and then once step 4 is completed you can compute the algorithm you'll implement to see which flows need to be installed or removed:

3. Disable the previous enabled pipeline, if there was any.
4. of_multi_table requests all flows from flow_manager API.

Reply:
Whenever applicable try to rely on the DB while avoiding to create new shared instance-level data instructures, unless absolutely needed. The reason why mef_eline and other NApps use to "load" a lot of things in memory were because back then it could rely on a full-fledged DB like Mongo.

Reply:
These change were made into another PR.